litmus-portal/frontend/src/views/CreateWorkflow/TuneWorkflow/AddProbe/index.tsx

```diff
@@ -0,0 +1,276 @@
+import { Modal, ButtonOutlined, ButtonFilled, InputField } from 'litmus-ui';
+import { MenuItem, Select, InputLabel } from '@material-ui/core';
+import { useTranslation } from 'react-i18next';
+import React from 'react';
+import useStyles from './styles';
+
+interface AddProbeProps {
+  addProbe: () => void;
+  handleClose: () => void;
+  open: boolean;
+}
+
+const AddProbe: React.FC<AddProbeProps> = ({ addProbe, handleClose, open }) => {
+  const classes = useStyles();
+  const { t } = useTranslation();
+
+  const [probeData, setProbeData] = React.useState({
+    expectedResponseCode: '',
+    interval: '',
+    intervalDelaySeconds: '',
+    pollingInterval: '',
+    probeName: '',
+    probeProperties: '',
+    probeType: '',
+    retry: '',
+    timeout: '',
+    url: '',
+  });
+
+  return (
+    <Modal
+      open={open}
+      onClose={handleClose}
+      width="60%"
+      className={classes.modal}
+      aria-labelledby="simple-modal-title"
+      aria-describedby="simple-modal-description"
+      modalActions={
+        <ButtonOutlined className={classes.closeButton} onClick={handleClose}>
+          &#x2715;
+        </ButtonOutlined>
+      }
+    >
+      <div className={classes.modal}>
+        <form onSubmit={addProbe} className={classes.form}>
+          <div className={classes.heading}>
+            {t('createWorkflow.tuneWorkflow.addProbe.heading')}
+            <strong>
+              {' '}
+              {t('createWorkflow.tuneWorkflow.addProbe.headingStrong')}
+            </strong>
+          </div>
+          <div className={classes.formField}>
+            <InputLabel htmlFor="probe-name" className={classes.formLabel}>
+              {t('createWorkflow.tuneWorkflow.addProbe.labels.probeName')}
+            </InputLabel>
+            <InputField
+              variant="primary"
+              width="50%"
+              id="probe-name"
+              type="text"
+              value={probeData.probeName}
+              onChange={(e) =>
+                setProbeData({ ...probeData, probeName: e.target.value })
+              }
+            />
+          </div>
+          <div className={classes.formField}>
+            <InputLabel className={classes.formLabel} htmlFor="probe-type">
+              {t('createWorkflow.tuneWorkflow.addProbe.labels.probeType')}
+            </InputLabel>
+            <Select
+              value={probeData.probeType}
+              className={classes.select}
+              variant="outlined"
+              onChange={(e) =>
+                setProbeData({
+                  ...probeData,
+                  probeType: e.target.value as string,
+                })
+              }
+              inputProps={{
+                id: 'probe-type',
+              }}
+            >
+              <MenuItem value="Http">
```

Comment:
Can we make them uniform either all pascal or lower case? (preferred lower case)
{ Http, Cmd, K8s, Prom} OR {http, cmd, k8s, prom}